### ISSUE TITLE
Branch point to gh-pages on user pages

### ISSUE THREAD

**Issue by CGarces** [jekyll/github-metadata#111]
After change my layouts to use i see the the url generated point to the `gh-pages` branch, than not exist.

My blog is a user blog and only contains `master` as relevant branch.

I can't see in the documentation how set up the branch

**Comment by benbalter** [comment]
We can't speak to the specifics of your local se up. If you [contact GitHub support](https://github.com/contact) they should be able to help you get things straightened out.

**Comment by CGarces** [comment]
Not looks like a github issue, fails on my local environment and my CI environment.

Is the branch used by the plugin harcoded?

This is my current workflow:

- I create a new branch with the modification at local environment ->link point to `gh-pages` branch (wrong)
- Push the changes and send the PR. The PR trigger the CI tasks at codeship.com->link point to `gh-pages` branch (wrong)
- Accept the PR, [the CI build is launched again](https://app.codeship.com/projects/129605/builds/28101537?pipeline=3d834d06-4009-45eb-922b-3265ee93c1eb) ->link point to `gh-pages` branch (wrong)
- The page is rebuild by github ->link point to `master` branch (Ok)

**Comment by benbalter** [comment]
@CGarces See https://github.com/jekyll/github-metadata/pull/110. This is a known issue with a fix awaiting testing. If you're able to test that pull request and it resolves your issue we can get it merged and released.

**Comment by CGarces** [comment]
After update the build to 2.9.2:

Jekyll build still generates the page pointing to `gh-pages`

but manually change the ` _config.yml` just works for me 

```
github:
  source:
    branch: master
```
Note that this workaround not was working on 2.9.1

**Comment by benbalter** [comment]
@CGarces it sounds like your initial issue was resolved with that workaround, correct?

**Comment by CGarces** [comment]
Yes, is fixed with the workaround, but I still don't know why the plugin generate the url with `gh-pages` as branch.
Feel free to fix the issue if is a duplicate of a existing one.
